frontend/src/metabase/user/components/LoginHistoryList.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+
+import LoginHistory from "metabase/entities/loginHistory";
+
+const LoginHistoryItemRow = item => {
+  return (
+    <tr>
+      <td>
+        <tt>{item.timestamp}</tt>
+      </td>
+      <td>{item.location && item.location.description}</td>
+      <td>
+        <tt>{item.device_description}</tt>
+      </td>
+      <td>{item.active ? "✅" : "❌"}</td>
```

Comment:
If my session is active does that mean it's my current session from the device I'm currently using? I'd also say we should use our own iconography rather than emoji.

Reply:
you can be logged in on more than one device at the same time, so the session for the device you're using will be active but it doesn't mean there won't be other active sessions as well